app/components/affiliate-unit.js

```diff
@@ -14,6 +15,8 @@ export default Component.extend(
 
     classNames: ['affiliate-unit'],
 
+    showAffiiateUnitDisclaimer: !exists('.watch-show__disclaimer'),
```

Comment:
When will this value be evaluated? Is it a possible race condition between the loading of the JS and this code?

Reply:
Are you asking when will `exists()` be evaluated? This was tested and worked in both dev and sandbox environments. Any edge cases I'm not thinking of?

Reply:
Did you test it on a slow connection? I'm wondering if there is a race condition where it evaluates this property before it shows up on the page.

Reply:
Tested both the big affiliate unit and post search results affiliate unit in Chrome under Slow 3G conditions and disclaimer gets rendered with the unit.